homeassistant/components/smartthings/sensor.py

```diff
@@ -360,3 +365,47 @@ def state(self):
             return three_axis[self._index]
         except (TypeError, IndexError):
             return None
+
+
+class SmartThingsPowerConsumptionReportSensor(SmartThingsEntity):
+    """Define a SmartThings Power Consumption Report."""
+
+    def __init__(self, device):
+        """Init the class."""
+        super().__init__(device)
+        self._name = "Power Consumption Report"
+
+    @property
+    def name(self) -> str:
+        """Return the name of the binary sensor."""
+        return f"{self._device.label} {self._name}"
+
+    @property
+    def unique_id(self) -> str:
+        """Return a unique ID."""
+        return f"{self._device.device_id}.{self._name}"
```

Comment:
```suggestion
        return f"{self._device.device_id}.powerConsumptionReport"
```

Reply:
Great!  These are the kinds of things that I am really trying to memorize and and keep track of as I continue to work on HA - and there are a lot of them!